cli/pack/pack.go

```diff
@@ -244,6 +251,54 @@ func setSDKPath(ctx *context.Ctx) error {
 	return nil
 }
 
+func fillScript(filename string, defaultScriptFilePath string, outputScript *string) error {
+	var err error
+
+	if filename == "" {
+		if _, err := os.Stat(defaultScriptFilePath); err == nil {
+			log.Debugf("Default script is used: %s", defaultScriptFilePath)
+			filename = defaultScriptFilePath
+		} else if !os.IsNotExist(err) {
+			return fmt.Errorf("Failed to use default script file: %s", err)
+		}
+	}
+
+	if filename != "" {
```

Comment:
It seems this ``if`` necessary.

Reply:
We need this `if` to check if the `filename` has changed to the default file name.

Reply:
Looks like at line 265 ``filename`` always not "".  I see code below and i think ``defaultScriptFilePath`` can't be a empty. Or I'm wrong?

Reply:
`filename` is empty if user did not specify one, if there is file with default name (`preinst.sh` for pre-install, `postinst.sh` for post-install) and user did not specify some file name, then at line 260 `filename` sets to `defaultScriptFilePath`